### PR TITLE
Modify README.md

### PR DESCRIPTION
**Pull request by paveg**
Hi, there. I improved README.md as follows.

- unification of style
- Attached Apache license label.

Please merge if possible.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+[![CircleCI](https://circleci.com/gh/kufu/activerecord-bitemporal.svg?style=svg)](https://circleci.com/gh/kufu/activerecord-bitemporal)
 
 ## Installation
 
 Add this line to your application's Gemfile:
 
-```ruby
-gem 'activerecord-bitemporal'
-```
+    $ gem 'activerecord-bitemporal'
```
Comment (meganemura) [inline README.md]: @paveg 
I think original one is better since this line is for Gemfile, not for shell.
Would you please undo this change?

Comment (paveg) [reply README.md]: I misunderstood🤦‍♂️
I'll change it.

**Comment by paveg** [comment]
I revert it 🙆‍♂️

**Comment by paveg** [comment]
I cannot merge it 🙏
